CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ Breaking changes:
   within that block inline.
   (PR [#607](https://github.com/alphagov/govuk-frontend/pull/607))
 
+Fixes:
```

Comment:
I think this is technically an internal change since we don't ship the yaml files.